docs: add progressive "Task Lists" sections to CONTRIBUTING.md (#22537)
vlang/v: CONTRIBUTING.md

> ## Contributing task lists

Reviewer: ``` suggestion
## Tasks to do, to help the V project
```

> 
> (suggested by [@spytheman](https://github.com/spytheman))

Reviewer: ```suggestion
```

The tasks in the lists below are ordered in terms of easiness/time/nerves
investment.

### Starting tasks

> 1. Read the docs.

Reviewer: ```suggestion
1. Read the [language documentation](https://docs.vlang.io/introduction.html) and [standard module documentation](https://modules.vlang.io/).
```

> 2. Fix errors in the docs, and make PRs about it.

Reviewer: ```suggestion
2. Fix doc errors and places you found unclear, and make [PRs](https://github.com/vlang/v/pulls) about it.
```

3. Make V programs, in the areas that you are interested in, or help others make
   V programs. If the programs/libraries are public, post them to the channel
   `#showcase-discussion` on [Discord](https://discord.gg/vlang) and/or to
   [awesome-v](https://github.com/vlang/awesome-v). You can also answer other
   people's questions in Discord channels `#help` or `#v-chat`, or in
   [github's discussions page](https://github.com/vlang/v/discussions).

4. Read the new issues in the tracker.

5. Try to reproduce them on your system, and comment in the issues, with the
   results.

6. Read the PRs, try to spot errors in them, and comment about them.

### Medium tasks (after gathering experience with 1..6)

7. Make PRs, with bug fixes to existing issues (found doing 1..6).

8. Suggest new features, tools, or modifications to the existing ones here,
   based on the experience, that you gathered doing 1..7.

9. Make PRs with implementations of your suggestions, or based on other people's
   suggestions, based on 1..7 and the feedback from 8.

### Advanced tasks (after doing 1..9 for a while)

> 10. work on [V RFCs](https://github.com/vlang/rfcs/) (submitting new ones,

Reviewer: ```suggestion
10. Work on [V RFCs](https://github.com/vlang/rfcs/) (submitting new ones,
```
